src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -314,5 +327,19 @@ private static void WriteResultTable(IEnumerable<Counter> counters, Logger logge
                 logger.Log($"{counter.Name,-15}|{average,-15}|{min,-15}|{max,-15}");
             }
         }
+
+        private static Dictionary<string, string> ParseStringToDictionary(string s)
+        {
+            Dictionary<string, string> dict = new Dictionary<string, string>();
```

Comment:
another good spot for `var`.
```suggestion
            var dict = new Dictionary<string, string>();
```